hrqb/tasks/employee_salary_history.py

```diff
@@ -58,10 +58,12 @@ def get_dataframe(self) -> pd.DataFrame:
                 [
                     row.mit_id,
                     row.position_id,
-                    row.appointment_begin_date,
-                    row.appointment_end_date,
-                    row.start_date,
-                    row.end_date,
+                    str(row.appointment_begin_date),
+                    str(row.appointment_end_date),
+                    str(row.hr_personnel_action),
+                    str(row.hr_action_reason),
+                    str(row.start_date),
+                    str(row.end_date),
```

Comment:
It was also learned that dates and/or reasons can be `NULL` coming from the data warehouse (again, maybe new?) and so stringify-ing them here just to be safe.